canonical-websites/www.ubuntu.com
# Updated alternative download's network installer text

## Done

* Minor updates to @cbrownstein's excellent PR #2605

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/download/alternative-downloads](http://0.0.0.0:8001/download/alternative-downloads)
- Make sure the 'Network installer' section makes sense and follows the [copy doc](https://docs.google.com/document/d/1VhmFVE3dK81mE3zcC9FVA6f5SGJ-DHan7WWlrlfRcvo/edit)


## Issue / Card

Fixes #2583
